spacy/pipeline/spancat.py

```diff
@@ -154,6 +186,67 @@ def make_spancat(
     )
```

Comment:
For the "old" factory, I would also fill in the "new" variables with their correct values so that it's robust to changing defaults in the constructor.

Reply:
Basically avoid relying on the default values for both constructor calls.

Reply:
ok that makes more sense!